std/build.zig

```diff
@@ -769,7 +769,7 @@ pub const Builder = struct {
                 if (code != 0) {
                     warn("The following command exited with error code {}:\n", code);
                     printCmd(null, argv);
-                    std.debug.panic("exec failed");
+                    std.os.exit(@truncate(u8, code));
```

Comment:
Haven't looked at all the code too much, but would it make more sense to return an error code instead?

Reply:
what do you mean by return an error code? here we just exit with the same error code as the spawned process.

Reply:
Something like this?
```
return error.ProcessFailed
```
Like I said I haven't looked at the code so I don't know who is calling this or if their are multiple callers that may want to handle failures differently.  If there's only one caller or they all just want to exit on error then this is fine.  Just asking the question.

Reply:
wouldn't that lead to an error trace instead of forwarding the error code from the spawned process?

Reply:
All it does is allow the caller to handle it.  If the caller doesn't handle it and it passes all the way back up to `bootstrap.zig` then yes it would result in a stack trace.  Which is why I qualified what I said with "I haven't looked at the code".  But maybe some callers want to handle it differently which is why you would return an error code.  But maybe there is only ever going to be one caller and they always want to exit, in which case this is fine.

Reply:
previously we panicked so i think it's safe to assume that we can exit here. if #2647 is passed returing an error along with the term code would be a lot nicer than exiting here imo.

Reply:
True.  It's an improvement.  Was just suggesting another improvement if it makes sense.